### DISCUSSION TITLE
Help with Zero touch

### DISCUSSION THREAD

**Discussion by Dnlogan87** [FreeTAKTeam/FreeTakServer#634] [FreeTAKServer Support]
Trying to install zero touch on AWS. Went through the steps to setup AWS and now onto FTS setup. When I get to the last command line "wget -qO - bit.ly/FTSzerotouch | sudo bash" the install initializes and fails immediately after checking for supported OS.
It appears to be requiring a "y" entry to continue with install but does not receive it and cancels. It gives me no opportunity to enter "y".I've included a photo for reference. I'm new to all of this and not immensely knowledgeable with Ubuntu so your help is greatly appreciated.
![IMG_20231018_174308_377](https://github.com/FreeTAKTeam/FreeTakServer/assets/148401589/7a084e18-b6c6-406c-ac35-86241603678f)

**Comment by jweaver72** [comment]
Zero touch requires 22.04. Have you tried using 22.04 LTS
